azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -113,14 +113,20 @@ def _create_mqtt_client(self):
 
         # Instantiate client
         mqtt_client = mqtt.Client(
-            client_id=self._client_id, clean_session=False, protocol=mqtt.MQTTv311
+            client_id=self._client_id, 
+            clean_session=False, 
+            protocol=mqtt.MQTTv311,
+            transport="websockets"
```

Comment:
>transport="websockets" [](start = 12, length = 22)

This file will obviously change later.  Specifically, it will send transport="tcp" if not using websockets and transport="websockets" only if using websockets.  The call to ws_set_options is only necessary if you're using websockets, and tcp and websockets both use different ports.